nncf/quantization/algorithms/accuracy_aware/utils.py

```diff
@@ -0,0 +1,78 @@
+"""
```

Comment:
I would suggest to rename `utils.py` -> `quantizer_removing.py` or something like that. Also I would suggest to move it to `nncf/common/quantization`.

cc' @vshampor, @KodiaqQ

Reply:
I fixed it.